dll/win32/shell32/CDefaultContextMenu.cpp

```diff
@@ -389,7 +395,7 @@ CDefaultContextMenu::LoadDynamicContextMenuHandler(HKEY hKey, REFCLSID clsid)
         return hr;
     }
 
-    hr = pExtInit->Initialize(m_pidlFolder, m_pDataObj, hKey);
+    hr = pExtInit->Initialize(m_pDataObj ? NULL : m_pidlFolder, m_pDataObj, hKey);
```

Comment:
Is `Initialize()` call with a NULL first parameter a valid thing, or is it so that it will fail the initialize in that case?

Reply:
Look at my screenshot above, Windows passes NULL.